### PR TITLE
Update microsoft-staffhub-to-be-retired.md

### PR DESCRIPTION
**Pull request by v-behow**
Updating Office 365 to Microsoft 365

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/expand-teams-across-your-org/shifts/microsoft-staffhub-to-be-retired.md [hunk 1]
```diff
@@ -142,9 +142,9 @@ We encourage you to review your existing team rosters on StaffHub and where poss
  
 #### What happens to StaffHub team members who aren't provisioned or whose status is inactive?
 
-The user's schedule will be moved to Shifts, however, the user won't be a member of the team or an Office 365 Group.
+The user's schedule will be moved to Shifts, however, the user won't be a member of the team or an Microsoft 365 Group.
```
Comment (JohanFreelancer9) [inline Teams/expand-teams-across-your-org/shifts/microsoft-staffhub-to-be-retired.md:145]:
```suggestion
The user's schedule will be moved to Shifts, however, the user won't be a member of the team or a Microsoft 365 Group.
```

**Comment by scanum** [comment]
@v-behow Please sign the CLA

**Comment by scanum** [comment]
@v-behow Please sign the CLA

**Comment by scanum** [comment]
@v-behow We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this PR.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**